src/Catty/ViewController/CatrobatTableViewController/CatrobatTableViewController.m

```diff
@@ -123,14 +125,8 @@ - (void)viewWillAppear:(BOOL)animated
     BOOL lockIphoneEnabeled = [self shouldLockIphoneInAppWithoutScenePresenter];
     [[UIApplication sharedApplication] setIdleTimerDisabled:(lockIphoneEnabeled)];
     [self.tableView deselectRowAtIndexPath:[self.tableView indexPathForSelectedRow] animated:YES];
-    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
-    if ((! [defaults objectForKey:kUserIsFirstAppLaunch] || [defaults boolForKey:kUserShowIntroductionOnLaunch]) && animated == NO) {
```

Comment:
Please always show the introduction if "kUserShowIntroductionOnLaunch" is true.

Reply:
done